hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -223,6 +226,20 @@ public RpcClient(Configuration conf, String omServiceId) throws IOException {
         OzoneConfigKeys.OZONE_NETWORK_TOPOLOGY_AWARE_READ_DEFAULT);
   }
 
+  @Override
+  public List<OMRoleInfo> getOmRoleInfos() throws IOException {
```

Comment:
If some user call's this API after a period of time the information in OMRoleInfo might be stale. If we expose here, I think we should contact OM and get this info again.

Reply:
Thank you @bharatviswa504 . Fixed it by calling getServiceList() api when getOmRoles is called.